controllers/controllers_test.go

```diff
@@ -118,4 +119,34 @@ var _ = Describe("Identity Controller", func() {
 			Expect(body.SmartMangement.IsEntitled).To(Equal(false), "smart_management.is_entitled expected to be false")
 		})
 	})
+
+	Context("When the account number is -1 or '' ", func() {
+		fakeResponse := SubscriptionsResponse{
+			StatusCode: 200,
+			Data:       []string{"foo", "bar"},
+			CacheHit:   false,
+		}
+
+		It("should give back a valid EntitlementsResponse with insights false", func() {
+			// testing with account number "-1"
+			rr, body, _ := testRequest("GET", "/", "-1", DEFAULT_ORG_ID, fakeGetSubscriptions(DEFAULT_ORG_ID, fakeResponse))
+			expectPass(rr.Result())
+			Expect(body.Insights.IsEntitled).To(Equal(false), "insights.is_entitled expected to be false")
+			Expect(body.Openshift.IsEntitled).To(Equal(true))
```

Comment:
I think the same thing that Asa thought.
Why do the explanation on one and not the others.

The explanation on true will only show if the result is false.
Explanation is used when the Equal fails.

I suspect that the explanations are not needed in any case. Can you purposely fail the test and see what the output looks like without an explanation? I suspect (but dont know) that it is probably descriptive enough. I bet it probably says something close to what you are already saying.

Reply:
If you do use it for the expected false case though... because the default output sucks... please use it for the expected true case to. Because the default output is likely going to suck in that case too.

Reply:
Yep, we don't need 'em! The provided explanation is descriptive enough.